tests/cli/test_compose_ext4-filesystem.sh

```diff
@@ -15,6 +15,10 @@ CLI="./src/bin/composer-cli"
 
 
 rlJournalStart
+    rlPhaseStart "SELinux operates in enforcing mode"
+        rlAssertEquals "$(getenforce)" "Enforcing"
```

Comment:
`rlAssertEquals "SELinux operates in enforcing mode" "$(getenforce)" "Enforcing"`, see:
https://github.com/beakerlib/beakerlib/wiki/man#rlassertequals

and you can place the assert right after "compose start" opening clause. No need for separate phase only for that.

Reply:
Thank you for pointing this out. Fixed.